modules/api/src/main/java/org/eclipse/xpanse/api/ZitadelAuthorizationApi.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.api;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.xpanse.modules.models.security.constant.TokenResponse;
+import org.eclipse.xpanse.modules.security.zitadel.ZitadelAuthorizationService;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+
+/**
+ * REST interface methods for credential management.
+ */
+@Slf4j
+@RestController
+@Profile("zitadel")
+@CrossOrigin
+public class ZitadelAuthorizationApi {
```

Comment:
should these APIs also be exposed in the OpenAPI or should we suppress it? 
Just to avoid confusion to anyone who looks at the API doc.